server/src/main/kotlin/org/javacs/kt/completion/Completions.kt

```diff
@@ -120,7 +120,6 @@ private fun indexCompletionItems(file: CompiledFile, cursor: Int, element: KtEle
         .query(partial, queryName, limit = MAX_COMPLETION_ITEMS)
         .asSequence()
         .filter { it.kind != Symbol.Kind.MODULE } // Ignore global module/package name completions for now, since they cannot be 'imported'
-        .filter { it.fqName.shortName() !in importedNames && it.fqName.parent() !in wildcardPackages }
```

Comment:
Without this, completions were only being shown from the current file that was compiled.